### PR TITLE
Watchlist

### PR DESCRIPTION
**Pull request by KylieStanley**
Added functionality to the method where we grab the clicked cards and append them to the watchlist. 

Watchlist card styling still mostly skeletons, and will eventually need some conditional logic still to prevent from adding the same card over and over again to watchlist.

Naming may need adjusted, not sure if clicked cards will be the most descriptive or not for the intended purpose.

@brittanystoroz @thatpamiam

image

### CODE REVIEW

File: src/WatchlistCard.js [hunk 1]
```diff
@@ -2,14 +2,15 @@ import React, { Component } from 'react';
...
+  constructor (props) {
+    super(props);
   }
   render() {
     return (
       <li className="watchlist-card">
-        
-        
+      <h3>{this.props.card.name}</h3>
+      <p>Season {this.props.card.season}:
```
Comment (ashleylevi) [inline src/WatchlistCard.js:12]: Awesome!
Next step: let's add some conditional rendering to the watchlist so the episode has the name of the tv show on it!

File: src/WatchlistCard.css [hunk 2]
```diff
@@ -1 +1,9 @@
-WatchlistCard.css
\ No newline at end of file
+.watchlist-card {
+  height: 110px;
```
Comment (ashleylevi) [inline src/WatchlistCard.css:2]: This is the perfect size for out watchlist cards. 
Next step: let's add a background image for the episodes in our watchlists.

File: src/App.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import './App.css';
 import Search from './Search.js';
 import { tvShow, spinOff, episodes } from './Data.js';
 import Carousel from './Carousel.js';
+import Watchlist from './Watchlist.js';
```
Comment (K-Rye) [inline src/App.js:6]: This looks great. Everything appears to be working correctly and I have no suggestions for changes. It looks like what we set out to do at this point.

**Comment by brittanydionigi** [comment]
Really nice clean and readable code so far, nice work.

**Pull request merged**